app/controllers/EpicComponentsController.scala

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+import actions.CommonActions.NoCacheAction
+import play.api.mvc.Action
+import play.api.mvc.AnyContent
+import play.api.mvc.Controller
+
+import scala.concurrent.duration._
+
+class EpicComponentsController extends Controller {
+
+  def inlinePayment: Action[AnyContent] = Action { implicit request =>
+    Cached(30.days.toSeconds.toInt)(Ok(views.html.epicComponents.inlinePayment()))
```

Comment:
I would set the cache time to `60 seconds` so we only have to deploy this app, rather than having to either purge the fastly cache, or make a change on client sites.